RNA-seq/01-qc_trim_quant.md

```diff
@@ -0,0 +1,244 @@
+This is a time and resource intensive step, so we have prepared the required output in case we need it._
+
+### Input files
+
+The raw data FASTQ files (`fastq.gz`) for this sample, SRR585570, are in `data/fastq/gastric_cancer/SRR585570`.
+The first two directories, `data/fastq`, tell us that these files are data and 
+The third directory `gastric_cancer` tells us which experiment or dataset these data are from. 
+We'll be working with an additional dataset later in the module, so this distinction will become more important.
+
+The use of the `SRR585570` folder might seem unnecessary because we are only processing a single sample here, but keeping files for individual samples in their own folder keeps things organized for multi-sample workflows.
+(You can peak ahead and look at the `data/quant/NB_cell_line` folder for such an example.)
+
+There is no "one size fits all" approach for project organization. 
+It's most important that it's consistent, easy for you and others to find the files you need quickly, and minimizes the likelihood for errors (e.g., writing over files accidentally).
+
+### Quality control with FastQC
+
+The first thing our script does is use [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) for quality control in command line mode.
+Here's a link to the FastQC documentation: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/
+
+**Let's take a look at some example reports from the authors of FastQC:**
+
+* [Good Illumina data example report](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/good_sequence_short_fastqc.html) from FastQC
+* [Bad Illumina data example report](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/bad_sequence_fastqc.html) from FastQC
+
+FastQC runs a series of quality checks on sequencing data and provides an HTML report. As the authors point out in the [docs](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/2%20Basic%20Operations/2.2%20Evaluating%20Results.html):
+
+> It is important to stress that although the analysis results appear to give a pass/fail result, these evaluations must be taken in the context of what you expect from your library.
+
+The [documentation for individual modules/analyses](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/) in FastQC is a great resource!
+
+To save time, our script only runs one FASTQ file for SRR585570 with the following command:
+
+```bash
+# In the interest of time, we'll run one of the fastq files through FastQC
+fastqc data/fastq/gastric_cancer/SRR585570/SRR585570_1.fastq.gz \
+	-o QC/gastric_cancer/fastqc_output
+```
+
+#### `-o`
+
+The `-o` flag allows us to specify where the output of FastQC is saved. 
+Note that this is saved in a different place than the raw data files and in a directory specifically for quality control information.
+
+For comparison, we've prepared a FastQC report for one of the sets of reads for another sample in the experiment.
```

Comment:
For comparison to what?

Reply:
`SRR585570_1` I can make this more explicit